src/provider/model/AppDirectory.ts

```diff
@@ -0,0 +1,23 @@
+import {Application} from "../../client/directory";
+import {injectable} from 'inversify';
+
+@injectable()
+export class AppDirectory {
+    private _directory: Application[] = require('../../../res/provider/app-directory.json');
```

Comment:
I may be wrong, but would this cause the app directory JSON itself (not just the URL) to be hard-coded inside the main bundle once webpacked? If so, is that intended and/or likely to change in the future?

Reply:
Yep, will be webpack-ed in to the provider.

Totally temporary, pretty much all of this file is. Also note how everything is async, even though the data is hard-coded - assuming "worst case" for now, and will decide how to handle the data fetching in a later story.